inc/widgets/largo-series-posts.php

```diff
@@ -28,6 +28,7 @@ function widget( $args, $instance ) {
 		$instance['title_link'] = get_term_link( (int) $instance['series'], 'series' );
 		$term = get_term( $instance['series'], 'series' );
 		$title = apply_filters( 'widget_title', $term->name, $instance, $this->id_base );
+		$thumb = isset( $instance['thumbnail_display'] ) ? $instance['thumbnail_display'] : 'small';
```

Comment:
The idea was to copy the functionality from one of our other widgets, so the `small` default is ok.

It will be a selected from a dropdown, so I doubt there would be an unexpected size value. @benlk Do you know if that's ever happened with the Largo Recent Posts widget that this was copied from?

Reply:
I don't know if it has ever happened. 

The template that this widget renders only has cases for `small`, `medium`, and `large`, so an invalid choice or the `none` option would result in no thumbnail displaying:

https://github.com/INN/largo/blob/67b4dedb252220515243756e96ed2033dd7f0d38/partials/widget-content.php#L9-L30